### DISCUSSION TITLE
disabled option for combobox

### DISCUSSION THREAD

**Discussion by jeevan-sg** [twilio-labs/paste#2291] [General] [answered]
Hi team, 
can you advise how to implement a disabled option for a combobox optionTemplate? Thanks
https://paste.twilio.design/components/combobox

**Comment by nkrantz** [comment, marked as answer]
Hi @jeevan-sg! Combobox doesn't currently support disabled options, but it's a feature we're planning to incorporate into our upcoming work on the component. The spec is currently being finalized and reviewed with the DSYS committee, and the next steps are design work and then engineering. That is scheduled to be part of the [May 3 release](https://paste.twilio.design/roadmap/#2022-05-03-candidate), but feel free to follow along with [this JIRA ticket](https://issues.corp.twilio.com/browse/DSYS-2412).

**Reply by jeevan-sg** [reply]
hi @nkrantz, has this feature released and available? Thanks

**Reply by SiTaggart** [reply]
Hi @jeevan-sg, no, due to the Reliability First initiative our roadmap got pretty severely impacted, unfortunately. We're looking more likely to be September now.

**Reply by jeevan-sg** [reply]
Thanks @SiTaggart.

**Reply by AleenaSA** [reply]
Hi @nkrantz any update on the release?

**Reply by nkrantz** [reply]
Hi @AleenaSA - the PR is up, [feel free to follow along here](https://github.com/twilio-labs/paste/pull/2660). This is a large PR that adds a Multiselect Combobox to the Combobox package, so it'll be in the review stage for a little while. You can expect to have it ready to go by [our next release on October 18](https://paste.twilio.design/roadmap/#2022-10-18).

**Reply by shleewhite** [reply]
[Paste Core v15.3.0](https://www.npmjs.com/package/@twilio-paste/core/v/15.3.0) is live and it includes the combobox updates! Please let us know if you have any more questions.
